pkgs/data/themes/adwaita-qt/default.nix

```
@@ -18,11 +28,13 @@ mkDerivation rec {
 
   buildInputs = [
     qtbase
+    qt5.qtx11extras
+    xorg.libxcb
```

Instead, you would use a conditional here:

```suggestion
  ] ++ lib.optionals stdenv.isLinux [
    xorg.libxcb
```

---

Thanks, applied